.github/workflows/build.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: nightly-2024-02-03
```

Comment:
Do we want this pinned to a specific version rather than the latest nightly

Reply:
Yes, until the issue gets fixed. I'll write a bpfman issue to track the curve25519-dalek issue and unwind once it gets fixed.

Reply:
Sounds good thanks!